SparkPost/2web2ui
# SA-1381: Update routes for new report builder

### What Changed
 - Updated nav links and routes for new report builder

### How To Test
 - Check that old reports links now redirect to the report builder
 - Check that the preset reports are set actively

### To Do
